sdk/eventhub/Directory.Build.props

```diff
@@ -24,6 +24,6 @@
     this needs to be set after the common properties import.
   -->
   <PropertyGroup Condition="$(MSBuildProjectName.StartsWith('Azure.')) and $(MSBuildProjectName.EndsWith('.Tests'))">
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>11.0</LangVersion>
```

Comment:
If we don't need to explicitly pin anymore, then we can just remove this section (L18 - L28).    There was something that caused issues with C# 10 where we had to pin to 9.

Reply:
Done